### PR TITLE
Devoxx France Article

### PR DESCRIPTION
**Pull request by karesti**

### CODE REVIEW

File: _posts/2024-04-29-back-from-devoxx.adoc [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+title: Infinispan in Devoxx France 2024
+permalink: /blog/:year/:month/:day/infinispan-devoxx-france
+date: '2024-04-29T00:00:00.000-00:00'
+author: karesti
+tags: [ "conferences", "java", "devoxx", "devoxxfr" ]
+---
+
+== Infinispan 15 in Devoxx France
+
+This month we assisted to one of the biggest developers conference in Europe: Devoxx Francem in Paris.
```
Comment (rigazilla) [inline _posts/2024-04-29-back-from-devoxx.adoc:12]:
```suggestion
This month we assisted to one of the biggest developers conference in Europe: Devoxx France in Paris.
```

File: _posts/2024-04-29-back-from-devoxx.adoc [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+layout: blog
+title: Infinispan in Devoxx France 2024
```
Comment (ryanemerson) [inline _posts/2024-04-29-back-from-devoxx.adoc:3]: Infinispan at Devoxx ...?

File: _posts/2024-04-29-back-from-devoxx.adoc [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+permalink: /blog/:year/:month/:day/infinispan-devoxx-france
+date: '2024-04-29T00:00:00.000-00:00'
+author: karesti
+tags: [ "conferences", "java", "devoxx", "devoxxfr" ]
+---
+
+== Infinispan 15 in Devoxx France
+
+This month we assisted to one of the biggest developers conference in Europe: Devoxx France in Paris.
```
Comment (ryanemerson) [inline _posts/2024-04-29-back-from-devoxx.adoc:12]: "This month we participated in one of the ..."

File: _posts/2024-04-29-back-from-devoxx.adoc [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+==== Infinispan
+During this presentation, Alexander Schwartz and Ryan Emerson delve into the latest
+version of Keycloak and the High Availability architecture that facilitates deploying
+two sites with Keycloak.
+
+[caption="Keycloak",link=/assets/images/blog/2024-devoxx-france/keycloak1.jpg]
+image::/assets/images/blog/2024-devoxx-france/keycloak1.jpg[Keycloak]
+
+One key component enabling this architecture is the
```
Comment (ryanemerson) [inline _posts/2024-04-29-back-from-devoxx.adoc:46]: "A key ..."

File: _posts/2024-04-29-back-from-devoxx.adoc [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+Infinispan Server, with its Cross-Site replication capabilities.
+
+Initially, this capability is realized through Active-Passive replication,
```
Comment (ryanemerson) [inline _posts/2024-04-29-back-from-devoxx.adoc:49]: "this capability is used to provide Active-Passive replication, however the roadmap includes plans for Active-Active replication in the near future".

File: _posts/2024-04-29-back-from-devoxx.adoc [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+cache values. Moreover, since Infinispan 15.0, it can be used as an embedding store to
+execute vector search with Langchain and Langchain4j.
+
+
+[caption="Hibernate Search",link=/assets/images/blog/2024-devoxx-france/hibernateSearch2.jpg]
+image::/assets/images/blog/2024-devoxx-france/hibernateSearch2.jpg[Hibernate Search]
+
+=== Les Cast Codeurs Podcast
+
+Mentioning here that Infinispan is indirectly part of Devoxx France and the Les Cast Codeurs podcast too.
```
Comment (ryanemerson) [inline _posts/2024-04-29-back-from-devoxx.adoc:86]: I'm not sure what you're trying to say here?

Comment (karesti) [reply _posts/2024-04-29-back-from-devoxx.adoc:86]: that i'm part of the podcast and devoxx france org

Comment (karesti) [reply _posts/2024-04-29-back-from-devoxx.adoc:86]: I'm the writer of the article, so I don't want o talk about me in 3rd person xDDD

Comment (karesti) [reply _posts/2024-04-29-back-from-devoxx.adoc:86]: We are not using 3rd person in all the article, are we ? I use "we" or Infinispan

Comment (ryanemerson) [reply _posts/2024-04-29-back-from-devoxx.adoc:86]: Maybe I'm too used to technical docs being written in 3rd person :sleepy: . It's just a blog, so let's keep it as is so it's more personal :+1:

**Comment by ryanemerson** [comment]
Thanks @karesti

**Pull request merged**